app/src/main/java/io/appium/settings/SettingsReceivers.java

```diff
@@ -0,0 +1,74 @@
+package io.appium.settings;
+
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.IntentFilter;
+import android.os.Build;
+import android.util.Log;
+
+import androidx.annotation.RequiresApi;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import io.appium.settings.receivers.AnimationSettingReceiver;
+import io.appium.settings.receivers.BluetoothConnectionSettingReceiver;
+import io.appium.settings.receivers.ClipboardReceiver;
+import io.appium.settings.receivers.DataConnectionSettingReceiver;
+import io.appium.settings.receivers.HasAction;
+import io.appium.settings.receivers.LocaleSettingReceiver;
+import io.appium.settings.receivers.LocalesReader;
+import io.appium.settings.receivers.LocationInfoReceiver;
+import io.appium.settings.receivers.MediaScannerReceiver;
+import io.appium.settings.receivers.NotificationsReceiver;
+import io.appium.settings.receivers.SmsReader;
+import io.appium.settings.receivers.UnpairBluetoothDevicesReceiver;
+import io.appium.settings.receivers.WiFiConnectionSettingReceiver;
+
+public class SettingsReceivers {
+    private static final String TAG = "APPIUM SERVICE";
+
+    @RequiresApi(api = Build.VERSION_CODES.KITKAT)
```

Comment:
AS required to add `@RequiresApi(api = Build.VERSION_CODES.KITKAT)` to use `|` in exception. This project itself sets `minSdkVersion 21`, so this 19 thing is not necessary technically.